api/loglines.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const Joi = require('joi');
+const Build = require('../models/build');
+
+const SCHEMA_PARAMS = Joi.object().keys({
+    id: Joi.reach(Build.base, 'id'),
+    name: Joi.reach(Build.getStep, 'name')
+}).label('URL Parameters');
+
+const SCHEMA_QUERY = Joi.object().keys({
+    from: Joi.number().integer()
+        .min(0).default(0)
+        .description('Starting Line Number')
+}).label('Query Parameters');
+
+const SCHEMA_LOGLINE = Joi.object().keys({
+    n: Joi
```

Comment:
So the emitter writes to the log service with the log line as `{t,m,s}` https://github.com/screwdriver-cd/launcher/blob/e5c4cb1048152ca20773cca62617fe36cc9f9194/screwdriver/emitter.go#L25-L29

Are we going to modify these values in the log service or something?